### PR TITLE
[R4R] Add retry to conda-forge

### PR DESCRIPTION
**Pull request by sodre**
This is the second dependency for the latest superset version.

xref: conda-forge/superset-feedstock#1

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by sodre** [comment]
@conda-forge/help-python, `retry` is ready for review. It is dependency 2/2 for conda-forge/superset-feedstock#1

File: recipes/retry/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pbr
+  run:
+    - python
+    - decorator>=3.4.2
```
Comment (xhochy) [inline recipes/retry/meta.yaml]: Please add a space between package name and version spec.
```suggestion
    - decorator >=3.4.2
```

File: recipes/retry/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - py>=1.4.26,<2.0.0
```
Comment (xhochy) [inline recipes/retry/meta.yaml]:
```suggestion
    - py >=1.4.26,<2.0.0
```

**Comment by sodre** [comment]
@xhochy, thanks I missed those. I wonder if they could be added to the linter...

**Pull request merged**